<launch>
    <arg name="pkg_name" value="auto_localization"/>
    <arg name="node_name" default="system_calibration_unit_test"/>

    <arg name="map" default="line_test" doc="Specify the map you wants"/>
    <arg name="output" default="testcircle" doc="Specify the output name of the new map"/>

        <remap from="complete_local_poses" to="duckietown3/system_calibration_gui/complete_local_poses" />
        <node name="system_calibration" pkg="$(arg pkg_name)" type="system_calibration.py"
              output="screen">
              <param name="~map" value="$(arg map)" type="str"/>
              <param name="~output_file" value="$(arg output)" type="str"/>
        </node>


</launch>
